From linux-usb-devel-admin@lists.sourceforge.net Thu Jan  5 11:46:21 2006
From: David Hollis <dhollis@davehollis.com>
To: Greg KH <gregkh@suse.de>
Cc: Charles Lepple <clepple@gmail.com>
Message-Id: <1136489989.2509.12.camel@dhollis-lnx.sunera.com>
Subject: USB: asix - Add device IDs for 0G0 Cable Ethernet
Date: Thu, 05 Jan 2006 14:39:49 -0500


Add device IDs for the 0G0 Cable Ethernet device as reported by
Charles Lepple <clepple@gmail.com>.

Signed-off-by: David Hollis <dhollis@davehollis.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/net/asix.c |    4 ++++
 1 file changed, 4 insertions(+)

--- gregkh-2.6.orig/drivers/usb/net/asix.c
+++ gregkh-2.6/drivers/usb/net/asix.c
@@ -916,6 +916,10 @@ static const struct usb_device_id	produc
 	// Linksys USB200M Rev 2
 	USB_DEVICE (0x13b1, 0x0018),
 	.driver_info = (unsigned long) &ax88772_info,
+}, {
+	// 0Q0 cable ethernet
+	USB_DEVICE (0x1557, 0x7720),
+	.driver_info = (unsigned long) &ax88772_info,
 },
 	{ },		// END
 };
